plugins/slowdrain.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: slowdrain
+spec:
+  version: v0.1.5
+  homepage: https://github.com/cturiel/kubectl-slowdrain
+  shortDescription: Drains a Kubernetes node deleting application pods one by one with a delay
+  description: |
+    This plugin drains a Kubernetes node by removing application pods one by one
+    with a configurable delay between each deletion. This is useful to avoid
+    application downtime when draining a node with a large number of pods.
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/cturiel/kubectl-slowdrain/releases/download/v0.1.5/kubectl-slowdrain_v0.1.5_linux_amd64.tar.gz
+    sha256: e96b3e92a7db3a317f4ec936b7a79b19aa4a7b01fc240d5dc0d640d481f7672e
+    bin: kubectl-slowdrain
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/cturiel/kubectl-slowdrain/releases/download/v0.1.5/kubectl-slowdrain_v0.1.5_darwin_amd64.tar.gz
+    sha256: 46497b4432da5c05f28f73cae01bb7403d002546fabceffef8b453c7022f19aa
+    bin: kubectl-slowdrain
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    uri: https://github.com/cturiel/kubectl-slowdrain/releases/download/v0.1.5/kubectl-slowdrain_v0.1.5_darwin_arm64.tar.gz
+    sha256: c89359a8c16c5f43d5fcd6dbed5a21fb4202409e979e43f2822d556af3b80bfc
+    bin: kubectl-slowdrain
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/cturiel/kubectl-slowdrain/releases/download/v0.1.5/kubectl-slowdrain_v0.1.5_windows_amd64.zip
+    sha256: 75ab886c4a0b5ef8ba2248c08e173356e75a4d28ba8c357fb6b21e9bc386f0ce
+    bin: kubectl-slowdrain.exe
+  caveats: |
+    Usage:
+      $ kubectl slowdrain <node-name> --delay 30
+
+    For additional options:
+      $ kubectl slowdrain --help
+      or https://github.com/cturiel/kubectl-slowdrain/blob/v0.1.5/doc/USAGE.md
```

Comment:
Please remove this. We show exactly this upon installation. This field is for actual caveats and important notes.